### PR TITLE
Updated Personnel to include Rank. Added Fleet Personnel Creation and…

### PR DESCRIPTION
**Pull request by bngweny**
Updated the Personnel table to include RankId
Added FleetPersonnel Table
Added basic update/delete statements

### CODE REVIEW

File: Statements/personnel.sql [hunk 1]
```diff
@@ -13,15 +13,15 @@ IF (EXISTS (SELECT *
...
-          ('Ello Asty', 188, 77, '57BBY', 'Male', 'Abednedo', 1),
-          ('Poe Dameron', 172, 80, '2ABY', 'Male', 'Yavin', 1),
-          ('Sila Kott', 172, 80, '23BBY', 'Female', 'Toprawa', 1)
+          ('Garven Dreis', 202, 136, '37BBY', 'Male', 'Virujansi', 1, 1),
+          ('Leia Organa', 150, 49, '19BBY', 'Female', 'Alderaan', 1, 1),
+          ('Biggs Darklighter', 183, 84, '19BBY', 'Male', 'Tatooin', 1, 1),
+          ('Wes Janson', 182, 77, '57BBY', 'Male', 'Taanab', 1, 1),
+          ('Ello Asty', 188, 77, '57BBY', 'Male', 'Abednedo', 1, 1),
+          ('Poe Dameron', 172, 80, '2ABY', 'Male', 'Yavin', 1, 1),
+          ('Sila Kott', 172, 80, '23BBY', 'Female', 'Toprawa', 1, 1)
```
Comment (JasP19) [inline Statements/personnel.sql:24]: Maybe we shouldn't assign the same rank to everyone. What do you think>

Comment (JasP19) [reply Statements/personnel.sql:24]: And the same for the faction

Comment (bngweny) [reply Statements/personnel.sql:24]: Yes definitely. I forgot to change it 😀

Comment (JasP19) [reply Statements/personnel.sql:24]: Awesome 😃

**Pull request merged**